core/src/common/context-options.ts

```diff
@@ -1,3 +1,7 @@
 export enum ContextOptions {
-  DISABLE_CONSTRUCTS_DEPLOYMENT_TRACKING = '@aws-analytics-reference-architecture/disableConstructsDeploymentTracking'
-}
\ No newline at end of file
+  DISABLE_CONSTRUCTS_DEPLOYMENT_TRACKING = '@aws-analytics-reference-architecture/disableConstructsDeploymentTracking',
+  EMR_EKS_TRACKING_ID = 'uksb-1sljh1bin',
+  DATA_ENG_PLATFORM_ID = 'uksb-1sljh1big',
+  DATA_LAKE_ID = 'uksb-1sljh1bja',
+  REPLAYER= 'uksb-1sljh1bgi',
```

Comment:
I would change to DATA_GENERATOR so it can be used to all the generator constructs